tools/js-sdk-release-tools/src/test/changelog/testCases/patch.1.new.rlc.api.md

```diff
@@ -0,0 +1,38 @@
+# patch
+
+``` ts
+export interface ClustersGet {
+    a: string;
+}
+
+export interface ClusterGetOld {
+    a: number;
+}
+
+export interface Routes {
+    (path: "basic", subscriptionId: string, resourceGroupName: string, clusterName: string): ClusterGetOld;
```

Comment:
do we plan to add cases for our meeting discussed?

Reply:
the only case need to add is the parameter order detection, right?

Reply:
how about any/unknow/null cases?

Reply:
Right. Let me summarize them in loop and add them in next pr once Jose confirm it

Reply:
any/unknow/null cases: yes
expand the scopes for union: no
let me add it in next pr